### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by alanpurple** [tensorflow/text#326]
build error for tensorflow 2.2.0,2.3.0-rc.0
master branch, with cuda 10.2

using anaconda with python 3.8.2
protobuf & libprotobuf 3.12.3

```
ERROR: /home/wmind/repo/tftext/tensorflow_text/core/kernels/BUILD:310:18: C++ compilation of rule '//tensorflow_text/core/kernels:sentencepiece_kernels' failed (Exit 1)
In file included from tensorflow_text/core/kernels/sentencepiece_kernels.cc:23:0:
bazel-out/k8-opt/bin/external/com_google_sentencepiece/src/sentencepiece.pb.h:17:2: error: #error This file was generated by an older version of protoc which is
 #error This file was generated by an older version of protoc which is
  ^~~~~
bazel-out/k8-opt/bin/external/com_google_sentencepiece/src/sentencepiece.pb.h:18:2: error: #error incompatible with your Protocol Buffer headers. Please
 #error incompatible with your Protocol Buffer headers. Please
  ^~~~~
bazel-out/k8-opt/bin/external/com_google_sentencepiece/src/sentencepiece.pb.h:19:2: error: #error regenerate this file with a newer version of protoc.
 #error regenerate this file with a newer version of protoc.
  ^~~~~
Target //oss_scripts/pip_package:build_pip_package failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 68.145s, Critical Path: 25.44s
INFO: 640 processes: 640 local.
FAILED: Build did NOT complete successfully
```

**Comment by broken** [comment]
TF recently upgraded the library of protobuf. We have not updated ours because our CI tests are against the stable version of TF that is using the previous version. We're investigating what changes we should make with core TF to prevent library upgrades like this burdening users that build from head.

In the meantime, updating the protobuf library in the WORKSPACE will fix this.

```
    tf_http_archive(
        name = "com_google_protobuf",
        patch_file = clean_dep("//third_party/protobuf:protobuf.patch"),
        sha256 = "cfcba2df10feec52a84208693937c17a4b5df7775e1635c1e3baffc487b24c9b",
        strip_prefix = "protobuf-3.9.2",
        system_build_file = clean_dep("//third_party/systemlibs:protobuf.BUILD"),
        system_link_files = {
            "//third_party/systemlibs:protobuf.bzl": "protobuf.bzl",
        },
        urls = [
            "https://storage.googleapis.com/mirror.tensorflow.org/github.com/protocolbuffers/protobuf/archive/v3.9.2.zip",
            "https://github.com/protocolbuffers/protobuf/archive/v3.9.2.zip",
        ],
    )
```

**Comment by alanpurple** [comment]
@broken 

thanks!!! it works

**Comment by broken** [comment]
We've updated and released 2.3.0-rc1. I believe all build errors should be resolved. Please reopen if you run into further trouble.
